src/views/catalog/templatescatalog/components/TemplatesCatalogDrawer/TemplatesCatalogDrawerCreateForm.tsx

```diff
@@ -29,173 +29,170 @@ import {
-                        data-test-id="vm-name-input"
-                        name="vmname"
-                        aria-label="virtualmachine name"
-                        value={vmName}
-                        onChange={setVMName}
-                      />
-                    </FormGroup>
-                  </SplitItem>
-                  <SplitItem>
-                    <DescriptionList>
-                      <DescriptionListGroup>
-                        <DescriptionListTerm>{t('Project')}</DescriptionListTerm>
-                        <DescriptionListDescription>{namespace}</DescriptionListDescription>
-                      </DescriptionListGroup>
-                    </DescriptionList>
-                  </SplitItem>
-                </Split>
+                        className="template-catalog-drawer-form-name"
+                        fieldId="vm-name-field"
+                      >
+                        <TextInput
+                          isRequired
+                          type="text"
+                          data-test-id="vm-name-input"
+                          name="vmname"
+                          aria-label="virtualmachine name"
+                          value={vmName}
+                          onChange={setVMName}
+                        />
+                      </FormGroup>
+                    </SplitItem>
+                    <SplitItem>
+                      <DescriptionList>
+                        <DescriptionListGroup>
+                          <DescriptionListTerm>{t('Project')}</DescriptionListTerm>
+                          <DescriptionListDescription>{namespace}</DescriptionListDescription>
+                        </DescriptionListGroup>
+                      </DescriptionList>
+                    </SplitItem>
+                  </Split>
+                </StackItem>
+                <StackItem />
+                <StackItem>
+                  <Checkbox
+                    id="start-after-create-checkbox"
+                    isChecked={startVM}
+                    onChange={(v) => setStartVM(v)}
+                    label={t('Start this VirtualMachine after creation')}
+                  />
+                </StackItem>
+              </>
+            ) : (
+              <StackItem>
+                {t(
+                  'This Template requires some additional parameters. Click the Customize VirtualMachine button to complete the creation flow.',
+                )}
               </StackItem>
-              <StackItem />
+            )}
+            <StackItem />
+            {quickCreateError && (
               <StackItem>
-                <Checkbox
-                  id="start-after-create-checkbox"
-                  isChecked={startVM}
-                  onChange={(v) => setStartVM(v)}
-                  label={t('Start this VirtualMachine after creation')}
-                />
+                <Alert variant="danger" title={t('Quick create error')} isInline>
```

Comment:
U can use AlertVariant from patternfly

Reply:
Hmm, what is the preferred component we should use and why? I don't see much difference, at least from the functional point of view, but maybe there is some difference...

Reply:
done thanks